commit 454eb6e46bace3b3142c1c62c08537ca88787afa (refs/changes/66/1766/1)
Author: Jordan Hendricks <jordan.hendricks@joyent.com>
Date:   2017-04-07T17:41:14+00:00 (2 years, 6 months ago)
    
    joyent/node-manta#304 Bump version to 4.3.0 for multipart upload changes

diff --git a/CHANGES.md b/CHANGES.md
index 32abfe4..a18a1b6 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -2,6 +2,10 @@
 
 ## not yet released
 
+## 4.3.0
+
+- MANTA-2169: Support multipart upload of a single file to Manta
+
 ## 4.2.0
 
 Minor bump due to relaxation of API requirements in `mfind` (NotFound
diff --git a/package.json b/package.json
index e9c48bb..6c131bc 100644
--- a/package.json
+++ b/package.json
@@ -7,7 +7,7 @@
         "type": "git",
         "url": "git://github.com/joyent/node-manta.git"
     },
-    "version": "4.2.0",
+    "version": "4.3.0",
     "main": "./lib/index.js",
     "dependencies": {
         "assert-plus": "^1.0.0",
